macos/FluentUI/Button/Button.swift

```diff
@@ -524,6 +524,7 @@ class ButtonCell: NSButtonCell {
 @objc(MSFButtonSize)
 public enum ButtonSize: Int, CaseIterable {
 	case large
+	case medium
```

Comment:
If we have large, medium, small, would it be worth it to consider changing to an objective size name for the enums, like we did for Avatar sizes? e.g., instead of medium, it would be size24?

Reply:
(I recognize this would be a breaking change, but may be better to do it sooner than later)

Reply:
i think we should hold off on public enum change until we have a solid fluent 2 mac design toolkit.